content/automate/ManageIQ/AutomationManagement/AutomationManager/Provisioning/Profile.class/_missing.yaml

```diff
@@ -0,0 +1,10 @@
+---
+object_type: instance
+version: 1.0
+object:
```

Comment:
Everything in Provisioning/Profile.class would all go away if we don't care about per ldap group statemachines...